### PR TITLE
Upgrade to 0.26

### LINKED ISSUE #104: Upgrade to 0.26
…when available. Requires #103.

### PR DESCRIPTION
**Pull request by lizconlan**
Sanity check review requested - have I missed anything before I start on the locale change?

(or more correctly, while I work on locale changes in a separate branch but before I smash the 2 things together)

Closes #104

### CODE REVIEW

**Comment by lizconlan** [comment]
Needs
 - [ ] check of functional overrides
 - [ ] locale change work

File: assets/stylesheets/responsive/custom.scss [hunk 1]
```diff
@@ -458,21 +458,16 @@ a.link_button_green_large {
 
 div.correspondence {
   background-color: $correspondence-bg;
-  padding-bottom: 0.5em;
-  .event_actions {
-    margin-bottom: 0;
```
Comment (garethrees) [inline assets/stylesheets/responsive/custom.scss]: This commit description isn't particularly useful.

Why is it "better"? What have we done and why?

**Comment by garethrees** [comment]
> have I missed anything before I start on the locale change?

Is there anything preventing us releasing 0.26 and _then_ making a locale change?

**Comment by lizconlan** [comment]
> Is there anything preventing us releasing 0.26 and then making a locale change?

Nope, that's a thing. Only advantage would be one deploy instead of two.

File: assets/stylesheets/responsive/custom.scss (hunk 1 above)
Comment (lizconlan) [inline assets/stylesheets/responsive/custom.scss]: Yeah that is a bit rubbish. Um... "Update CSS to use the new request page layout" would probably be better (I'll check - and split if necessary - then update)

Comment (lizconlan) [inline assets/stylesheets/responsive/custom.scss]: (It discards the use of Courier for incoming messages and - incompletely - allows the new link to this layout to work)

**Comment by garethrees** [comment]
> > Is there anything preventing us releasing 0.26 and then making a locale change?
>
> Nope, that's a thing. Only advantage would be one deploy instead of two.

Okay lets do that then, so that the upgrade isn't held up by any extra complications with a locale change.

**Comment by lizconlan** [comment]
Code as reviewed at this point. Rebased onto master and the "Better CSS" commit renamed to "Update css for new request page layout"

**Comment by lizconlan** [comment]
> Follower count should have a border and action menu should be a secondary button as per WDTK:
>
> screen shot 2017-01-11 at 10 28 39

For consistency, similar tweak required for the Authority page

WDTK screenshot:

screen shot 2017-01-11 at 13 33 37

**Comment by lizconlan** [comment]
ah, this looks slightly odd on a white background:

screen shot 2017-01-11 at 15 01 15

(as opposed to)

screen shot 2017-01-11 at 15 02 09

[space left blank for own "missing the point" gag]

**Comment by garethrees** [comment]
@wrightmartin might have a quick fix, but I don't think this should prevent us getting this released if its going to take any length of time to investigate. We can always ticket it and fix after release.

**Comment by lizconlan** [comment]
New link to this button:

screen shot 2017-01-11 at 17 44 01

screen shot 2017-01-11 at 17 44 25

(A little cheating to hardcode the border colour as using the theme light grey didn't look right)

Fixed request header date:

screen shot 2017-01-11 at 17 45 02

screen shot 2017-01-11 at 17 46 02

Action menu buttons:

screen shot 2017-01-11 at 17 47 15

Authority page buttons:

screen shot 2017-01-11 at 17 47 59

(in the interests of not confusing myself, that's an unfollow button)

**Comment by garethrees** [comment]
Looks good! I made a couple of tweaks in 3b90065 to set the follow button to the primary colour as its now the primary CTA. I also got rid of a few unnecessary styles to clear out some noise. Cheers!

**Pull request merged**